.github/workflows/build-test-release.yml

```diff
@@ -15,11 +15,13 @@ jobs:
             - name: Checkout Repo
               uses: actions/checkout@v3
```

Comment:
Let's upgrade this to v4 as well? same for the other pipelines

Reply:
Updated